tests/Licenses/LicensesClientTest.php

```diff
@@ -925,6 +946,73 @@ public function testGetConfigs(): void
         self::assertSame('pending', $config->getState());
     }
 
+    public function testGetPredictionRaw(): void
+    {
+        $this->httpClient
+            ->expects(self::once())
+            ->method('request')
+            ->with(
+                'get',
+                'https://www.test.com/licenses/XSP1234/predictions/daily',
+                [
+                    'headers' => [
+                        'apiKey' => '123456',
+                        'Content-Type' => 'application/json',
+                    ],
+                ]
+            )
+            ->willReturn(new Response(200, [], 'OK USA'));
+
+        $this->client->getPredictionRaw('XSP1234');
+    }
+
+    public function testGetPrediction(): void
+    {
+        $response = json_encode([
+            "status" => 200,
+            "data"=>
+                [
+                    "currency"=> "EUR",
+                    "updatedAt"=> "2022-10-26",
+                    "licenseRef"=> "XSP264509",
+                    "prediction"=> [
```

Comment:
this payload does not look like the one on your specs.yaml
```suggestion
                    "licenseReference"=> "XSP264509",
                    "predictions"=> [
```